tests/test_utils.py

```diff
@@ -306,3 +313,513 @@ def check_multistream_classic(fnames):
+                            skiprows=2,
+                            sep='\t')
+
+    # remove spaces from column names
+    vic_5 = vic_5.rename(columns=lambda x: x.replace(' ', ''))
+
+    # add datetime object column to snotel DataFrame
+    vic_5['DATES'] = pd.to_datetime(vic_5.YEAR * 10000 +
+                                            vic_5.MONTH * 100 +
+                                            vic_5.DAY,
+                                            format='%Y%m%d')
+
+    # remove year, day columns of DataFrame
+    vic_5.drop(['YEAR', 'DAY'], inplace=True, axis=1)
+
+    if ('HOUR' not in vic_5) and ('SEC' in vic_5):
+        # add hour column for groupby
+        vic_5['HOUR'] = (vic_5['SEC'] * (1/3600)
+                                        ).astype(int)
+
+    vic_5.index = vic_5.DATES
+
+    return(vic_5)
+
+def plot_science_tests(driver, test_type, science_test_data_dir, result_dir,
+                        plot_dir, plots_to_make, compare_data):
+
+    ''' makes science test figures
+
+    Parameters
+    ----------
+    driver: <str>
+        Name of Driver
+    test_type: <str>
+        Name of test
+    science_test_data_dir: <str>
+        Science test data directory
+    result_dir: <str>
+        Result directory
+    plot_dir: <str>
+        Directory for output plots
+    plots_to_make <Dict>
+        Keys that indicate which plots should be made
+    compare_data <Dict>
+        Keys that indicate which datasets and model output to use for comparison
+
+    Returns
+    ----------
+    '''
+    if test_type == "science_test_snotel":
+        plot_snotel_comparison(driver,
+                                science_test_data_dir,
+                                compare_data,
+                                result_dir,
+                                plot_dir,
+                                plots_to_make)
+
+    elif test_type == "science_test_fluxnet":
+        plot_fluxnet_comparison(driver,
+                                science_test_data_dir,
+                                compare_data,
+                                result_dir,
+                                plot_dir,
+                                plots_to_make)
+    else:
+        print("this has not been implemented in the VIC 5.0 science test suite")
+
+def plot_snotel_comparison(driver, science_test_data_dir,
+                            compare_data_dict,
+                            result_dir, plot_dir,
+                            plots_to_make):
+    ''' makes snotel figures
+
+    '''
+
+    # plot settings
+    plot_variables = ['OUT_SWE', 'OUT_ALBEDO', 'OUT_SALBEDO', 'OUT_SNOW_DEPTH',
+                    'OUT_SNOW_CANOPY', 'OUT_SNOW_PACK_TEMP', 'OUT_SNOW_MELT',
+                    'OUT_R_NET', 'OUT_LATENT', 'OUT_SENSIBLE']
+
+    plot_units = ['mm', 'fraction', 'fraction', 'mm', '%', 'degrees C',
+                    'mm', '$W/{m^2}$', '$W/{m^2}$', '$W/{m^2}$']
+    lw = 4.0
+    sns.set_style('whitegrid')
```

Comment:
that is also fine but that is not how this function works.  If you want to sandbox the style behavior, you'll you need to use the `with sns.set_style():` syntax for each figure:

http://web.stanford.edu/~mwaskom/software/seaborn/tutorial/aesthetics.html#temporarily-setting-figure-style